packages/generator/templates/app/package.json

```diff
@@ -47,7 +47,7 @@
     "@typescript-eslint/eslint-plugin": "2.x",
     "@typescript-eslint/parser": "2.x",
     "babel-eslint": "10.x",
-    "eslint": "6.x",
+    "eslint": "7.x",
```

Comment:
hi @flybayer you meant this change?

Reply:
Yes, this is good, but we also need to update the other `eslint-*` packages in this file

Reply:
Do you think it would be also worth updating?:
```
 @typescript-eslint/eslint-plugin                           2.x  →      3.x
 @typescript-eslint/parser                                  2.x  →      3.x
```

Reply:
yes! All the eslint related packages

Reply:
done, thanks!